monitoring/oplog-populator/alerts.yaml

```
@@ -36,6 +39,18 @@ groups:
       description: "Oplog populator failed to configure connector"
       summary: "Oplog populator couldn't update kafka connect connector"
 
+  - alert: KafkaConnectTooManyConnectors
+    Expr: |
+        sum(s3_oplog_populator_connectors{job="${oplog_populator_job}",namespace="${namespace}"})
+        > ${oplogPopulatorChangeStreamLimit}
+    For: "5m"
+    Labels:
+     severity: warning
+    Annotations:
+      zenko_service: backbeat-oplog-populator
+      description: "Kafka connector count is above the soft limit"
+      summary: "Kafka connector count is above the soft limit. If the number of buckets is too high, consider using a single replica for your cluster"
```

this is the other way round. `Summary` is a (short) summary, while `Description` should give the details.
```suggestion
      summary: "Kafka connector count is above the soft limit"
      description: "Kafka connector count is above the soft limit. If the number of buckets is too high, consider using a single replica for your cluster"
```